openfisca_core/scripts/__init__.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+import importlib
+import logging
+import sys
+
+from openfisca_core.reforms import Reform
+
+log = logging.getLogger(__name__)
+logging.basicConfig(format='%(levelname)s: %(message)s')
+
+
+def handle_error(error_message):
+    log.error(error_message)
+    sys.exit(1)
+
+
+def add_tax_benefit_system_arguments(parser):
+    parser.add_argument('-c', '--country_package', action = 'store', help = 'country package to use. If not provided, an automatic detection will be attempted by scanning the python packages installed in your environment which name contains the word "openfisca".')
+    parser.add_argument('-e', '--extensions', action = 'store', help = 'extensions to load', nargs = '*')
+    parser.add_argument('-r', '--reforms', action = 'store', help = 'reforms to apply to the country package', nargs = '*')
+
+    return parser
+
+
+def build_tax_benefit_sytem(country_package_name, extensions, reforms):
+
+    if country_package_name:
+        try:
+            country_package = importlib.import_module(country_package_name)
+        except ImportError:
+            handle_error('Could not import module `{}`. Make sure it is installed in your environment.'.format(country_package_name))
```

Comment:
All the `handle_error` strings should be unicode-strings (have a `u''`)

Otherwise, any unicode-string mentioned in `.format()` arguments will certainly provoke a `UnicodeDecodeError`

Remember that in Python2 using unicode-strings is the default. Encoded-string (UTF-8) happens only when outputting the string to the external world (terminal, network request, etc.)